### PR TITLE
Feat: Modified Garagitgit.cpp

### PR DESCRIPTION
**Pull request by strange-tiger**

### CODE REVIEW

**Review by Hyemin75 — APPROVED** [review]
great~!!!

**Review by csy-59 — APPROVED** [review]
Great!

**Review by seagull30 — APPROVED** [review]

File: Garagitgit/Garagitgit/Garagitgit.cpp [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 int main()
 {
-    std::cout << "Hello World!\n";
+    std::cout << "Hello Colllaboration\n";
```
Comment (ghost) [inline Garagitgit/Garagitgit/Garagitgit.cpp:7]: Good!!!

**Review by ghost — APPROVED** [review]
진짜 잘했어요~!

**Pull request merged**